src/node_dependencies/special_cases.js

```diff
@@ -0,0 +1,33 @@
+const { getPackageJson } = require('./package_json')
+
+const getPackageJsonIfAvailable = async (srcDir) => {
+  try {
+    const packageJson = await getPackageJson(srcDir)
+
+    return packageJson
+  } catch (_) {
+    return {}
+  }
+}
+
+const getModulesForNextJs = ({ dependencies }) => {
+  const externalModules = dependencies.next ? ['critters', 'nanoid'] : []
```

Comment:
> Should we check devDependencies too?

I'll add that. 👍

> Also since we want to exclude critters and nanoid from the bundle, should we use ignoredModules

My understanding is that Next.js does use `critters` in certain circumstances. If we add it to `ignoredModules`, its `require` will be left in place but the source will not be added to the bundle, so the `require` call will always throw.

Reply:
Good point - I'm still under the thought this will be configured externally.